packages/website/docs/getting-started.md

```diff
@@ -64,41 +87,41 @@ Make sure to provide a container (e.g., a `div`), not an `input`. Autocomplete g
 ```js title="JavaScript"
 import { autocomplete } from '@algolia/autocomplete-js';
 
+import '@algolia/autocomplete-theme-classic';
+
 autocomplete({
   container: '#autocomplete',
-  placeholder: 'Search the autocomplete documentation',
-  openOnFocus: true,
+  placeholder: 'Search for products',
   getSources() {
     return [];
   },
 });
 ```
 
-You may have noticed three new options: [`placeholder`](autocomplete-js#placeholder), [`openOnFocus`](autocomplete-js#openonfocus), and [`getSources`](sources#getsources).
-
-The [`placeholder`](autocomplete-js#placeholder) option defines the placeholder text to show until the user starts typing in the input, while the [`openOnFocus`](autocomplete-js#openonfocus) option determines whether to open the panel on [focus](https://developer.mozilla.org/en-US/docs/Web/API/Window/focus_event) or not, even when there's no query. It defaults to `false`, so you need to set it to `true` if you want the dropdown to appear as soon as a user clicks on it.
+You may have noticed two new options: [`placeholder`](autocomplete-js#placeholder) and [`getSources`](sources#getsources). The [`placeholder`](autocomplete-js#placeholder) option defines the placeholder text to show until the user starts typing in the input.
 
 Autocomplete is now plugged in. But you won't see anything appear until you define your [sources](sources).
 
 ## Defining what items to display
 
 [Sources](sources) define where to retrieve the items to display in your autocomplete dropdown. You define your sources in the [`getSources`](sources#getsources) function by returning an array of [source objects](sources#source). Each source object needs to include a [`getItems`](sources#getitems) function that returns the items to display. Sources can be static or dynamic.
 
-This example uses the [Algolia index](https://www.algolia.com/doc/faq/basics/what-is-an-index/) [powering the documentation search](https://docsearch.algolia.com/) on this site as a source. The [`autocomplete-js`](autocomplete-js) package provides a built-in [`getAlgoliaHits`](getAlgoliaHits) function for just this purpose.
+This example uses the [Algolia index](https://www.algolia.com/doc/faq/basics/what-is-an-index/) of [e-commerce products](https://github.com/algolia/datasets/tree/master/ecommerce) as a source. The [`autocomplete-js`](autocomplete-js) package provides a built-in [`getAlgoliaHits`](getAlgoliaHits) function for just this purpose.
 
 ```js title="JavaScript"
 import algoliasearch from 'algoliasearch/lite';
 import { autocomplete, getAlgoliaHits } from '@algolia/autocomplete-js';
 
+import '@algolia/autocomplete-theme-classic';
+
 const searchClient = algoliasearch(
-  'BH4D9OD16A',
-  'a5c3ccfd361b8bcb9708e679c43ae0e5'
+  'latency',
+  '6be0576ff61c053d5f9a3225e2a90f76'
 );
 
 autocomplete({
   container: '#autocomplete',
-  placeholder: 'Search documentation',
-  openOnFocus: true,
+  placeholder: 'Search for products',
   getSources({ query }) {
     return [
       {
```

Comment:
`sourceId` is required now.

```suggestion
      {
        sourceId: 'products',
```

(This applies to other occurrences.)

Reply:
https://github.com/algolia/autocomplete.js/pull/446/commits/7cee8896aeb8999328bc0846b3de27a856d7c22e